helm-cluster-values/fleet.yaml

```diff
@@ -22,4 +22,7 @@ helm:
     annotations:
       app: fleet
       more: data
-    replicaCount: ${ randNumeric 0 }
+      # to test the ability to use functions
+      randomNumber: ${ randNumeric 3 | quote }
+    # to test explicit null values
+    replicaCount: ~
```

Comment:
Now we don't test anymore if a template function can return null. Maybe we don't need to since https://github.com/rancher/fleet/blob/7b60b9199a3f636962e81f05bf8baef4754e9b59/internal/cmd/controller/target/target.go#L648 should be safe?

Reply:
It never returned null, isn't it? `randNumeric 0` returns an empty string, as explained in the description.